src/test/ui/or-patterns/or-patterns-syntactic-fail.rs

```diff
@@ -17,26 +17,5 @@ fn no_top_level_or_patterns() {
     fn fun1(A | B: E) {} //~ ERROR an or-pattern parameter must be wrapped in parenthesis
 
     fn fun2(| A | B: E) {}
-    //~^ ERROR a leading `|` is not allowed in a parameter pattern
-    //~| ERROR an or-pattern parameter must be wrapped in parenthesis
-}
-
-// We also do not allow a leading `|` when not in a top level position:
-
-fn no_leading_inner() {
```

Comment:
All of these cases should probably be kept to make sure that the leading `|`s indeed parse.

UPD: Or all of these are covered by `remove-leading-vert.rs`?

Reply:
I will take a closer look, but I think remove-leading-vert does indeed cover all of them...

Reply:
Indeed, they all appear to be covered by remove-leading-vert.rs